### PR TITLE
remove _pickAll

### PR DESCRIPTION
**Pull request by davidchambers**

### CODE REVIEW

File: dist/ramda.js [hunk 1]
```diff
@@ -2659,7 +2645,14 @@
      *      R.pickAll(['a', 'd'], {a: 1, b: 2, c: 3, d: 4}); //=> {a: 1, d: 4}
      *      R.pickAll(['a', 'e', 'f'], {a: 1, b: 2, c: 3, d: 4}); //=> {a: 1, e: undefined, f: undefined}
      */
-    var pickAll = _curry2(_pickAll);
+    var pickAll = _curry2(function pickAll(names, obj) {
+        var result = {};
+        for (var idx = 0, len = names.length; idx < len; idx += 1) {
```
Comment (buzzdecafe) [inline dist/ramda.js]: `while` ftw

Comment (davidchambers) [inline dist/ramda.js]: #886

Comment (davidchambers) [inline dist/ramda.js]: I've switched to a `while` loop. This pull request should no longer be controversial.

**Comment by buzzdecafe** [comment]
:cow:

**Pull request merged**